go-controller/pkg/crd/nodednsinfo/v1/register.go

```diff
@@ -0,0 +1,29 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var (
+	GroupName          = "k8s.ovn.org"
```

Comment:
argh, we've been doing this wrong. The same group is supposed to go in the same package / folder.